Azure pipeline error, Install Safari Technology Preview "no download for 10.16"

@foolip , @jcscottiii , @gsnedders , @whimboo 
Dear github admins and colleagues,

I am running again into this error in [PR34925](https://github.com/web-platform-tests/wpt/pull/34925) , just like in [issue 34752](https://github.com/web-platform-tests/wpt/issues/34752)
{
Check failure on line 30 in Build log
@azure-pipelines azure-pipelines / Azure Pipelines
Build log #L30
Bash exited with code '1'.

View raw log

Starting: Install Safari Technology Preview

Task         : Command line
Description  : Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
Version      : 2.201.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/command-line

Generating script.
 Starting Command Output 
/bin/bash --noprofile --norc /Users/runner/work/_temp/2aebd30f-ed7f-43f3-8a3d-c1f902896ae4.sh
+ ./wpt install --channel preview --download-only -d . --rename STP safari browser
INFO:install:Now installing safari browser...
INFO:install:Finding STP download URLs
INFO:install:Found STP URLs for macOS ==13.*, ~=12.3
Traceback (most recent call last):
  File "/Users/runner/work/1/s/./wpt", line 10, in <module>
    wpt.main()
  File "/Users/runner/work/1/s/tools/wpt/wpt.py", line 220, in main
    rv = script(*args, **kwargs)
  File "/Users/runner/work/1/s/tools/wpt/install.py", line 89, in run
    install(browser, kwargs["component"], destination, channel, logger=logger,
    raise ValueError(f"no download for {system_version}")
ValueError: no download for 10.16
##[error]Bash exited with code '1'.
Finishing: Install Safari Technology Preview
}

--- a/tools/ci/azure/install_safari.yml
+++ b/tools/ci/azure/install_safari.yml
@@ -9,6 +9,7 @@ steps:
 - ${{ if eq(parameters.channel, 'preview') }}:
   - script: |
       set -eux -o pipefail
+      export SYSTEM_VERSION_COMPAT=0
       ./wpt install --channel preview --download-only -d . --rename STP safari browser
       sudo installer -pkg STP.pkg -target LocalSystem
       # Workaround for `sudo safardriver --enable` not working on Catalina:

